lib/index.js

```diff
@@ -164,3 +167,45 @@ const findNodesForFilter = () => {
 const removeFoodRow = () => {
   event.target.closest('article').remove()
 }
+
+const calorieSorter = (originalOrder) => {
+  var count = 0
```

Comment:
This count is the click count, and it is used to determine which order the table should be sorted in.